EIPS/eip-1355.md

```diff
@@ -22,7 +22,7 @@ Provide minimal set of changes to Ethash algorithm to hinder and delay the adopt
    ```
    where `FNV1A_PRIME` is 16777499 or 16777639.
 2. Change the hash function that determines the DAG item index in Ethash algorithm from `fnv()` to new `fnv1a()`.
-   In [Main Loop](https://github.com/ethereum/wiki/wiki/Ethash#main-loop) change
+   In [Main Loop](https://ethereum.org/en/developers/docs/consensus-mechanisms/pow/mining/mining-algorithms/ethash/#main-loop) change
```

Comment:
Instead of linking here, it would be better to link to [EELS](https://github.com/ethereum/execution-specs/blob/forks/osaka/src/ethereum/ethash.py). We have instructions for these kinds of links in EIP-1.

That said, this proposal is Withdrawn, and likely doesn't need to be updated.